custom_components/tahoma/binary_sensor.py

```diff
@@ -89,13 +99,13 @@ def icon(self) -> Optional[str]:
         """Return the icon to use in the frontend, if any."""
         if self.device_class == DEVICE_CLASS_WATER:
             if self.is_on:
-                return "mdi:water"
+                return ICON_WATER
             else:
-                return "mdi:water-off"
+                return ICON_WATER
 
         icons = {
-            DEVICE_CLASS_GAS: "mdi:waves",
-            DEVICE_CLASS_RAIN: "mdi:weather-rainy",
+            DEVICE_CLASS_GAS: ICON_WAVES,
+            DEVICE_CLASS_RAIN: ICON_WATER_OFF,
```

Comment:
`ICON_WATER_OFF` is another icon?

Reply:
hmm not sure what I did there, probably a bad copy and paste

Reply:
fixed